src/Microsoft.ML.Data/Transforms/NormalizeUtils.cs

```diff
@@ -52,17 +52,19 @@ public interface IColumnAggregator<T>
         void Finish();
     }
 
-    internal interface IColumnFunction : ICanSaveModel
+    public abstract class IColumnFunction : ICanSaveModel
```

Comment:
>ICanSaveModel [](start = 44, length = 13)

can we implement `ICanSaveModel` explicitly?

Reply:
there is no communalities between them.. 

Reverting this back to an interface.  

---
In reply to: [232814114](https://github.com/dotnet/machinelearning/pull/1594#discussion_r232814114) [](ancestors = 232814114)